src/bin/verifbench.rs

```diff
@@ -0,0 +1,132 @@
+// -i, --inputs <inputs>    Sets number of public inputs
+// -p, --proofs <proofs>    Sets number of proofs in a batch
+// --nogpu                  Disables GPU
+
+use ff::Field;
+use rand::{thread_rng, Rng};
+use std::sync::Arc;
+
+use bellperson::groth16::{
+    prepare_batch_verifying_key, verify_proofs_batch, Parameters, Proof, VerifyingKey,
+};
+use clap::{App, Arg};
+use groupy::CurveProjective;
+use paired::bls12_381::Bls12;
+use paired::Engine;
+use std::time::Instant;
+
+fn random_points<C: CurveProjective, R: Rng>(count: usize, rng: &mut R) -> Vec<C::Affine> {
+    const DISTINT_POINTS: usize = 100;
+    (0..DISTINT_POINTS)
+        .map(|_| C::random(rng).into_affine())
+        .collect::<Vec<_>>()
+        .into_iter()
+        .cycle()
+        .take(count)
```

Comment:
Does it matter if `count` is greater than `DISTINCT_POINTS`?

Reply:
No in any case it'll be fine. I limited the number of distinct points because generating random points is very time consuming, so it's better to just repeat them.